docs/section21/Rule21-11A.md

```diff
@@ -0,0 +1,44 @@
+
+# Boiler - Rule 21-11A  
+
+**Rule ID:** 21-11A  
+**Rule Description:** When the system uses boilers the hot water system shall be modeled as primary only.  
+**Rule Assertion:** B-RMR  
```

Comment:
based on @mkarpman comment, the rule description would now become 'When the system uses boilers or purchased hot water, the hot water system shall be modeled as primary only'

Reply:
Updated.